cmd/vpp-agent-ctlv2/json/memif.json

```diff
@@ -4,12 +4,10 @@
     "ip_addresses": [
         "192.168.42.1/24"
     ],
-    "link": {
-        "memif": {
-            "id": 1,
-            "master": true,
-            "socket_filename": "/tmp/memif1.sock"
-        }
+    "memif": {
+        "id": 1,
+        "master": true,
+        "socket_filename": "/tmp/memif1.sock"
     },
     "name": "memif1",
     "type": 4
```

Comment:
We might use this opportunity to convert enums from ints to strings for more clarify and to make these data examples more readable to newcomers.